src/libs/OptionsListUtils.js

```diff
@@ -851,7 +851,9 @@ function getNewChatOptions(reports, personalDetails, betas = [], searchValue = '
  */
 
 function getShareDestinationOptions(reports, personalDetails, betas = [], searchValue = '', selectedOptions = [], excludeLogins = [], includeOwnedWorkspaceChats = true) {
-    return getOptions(reports, personalDetails, {
+    // We want to filter out any IOUs or expense reports
+    const filteredReports = _.filter(reports, (report) => !ReportUtils.isMoneyRequestReport(report));
```

Comment:
@fedirjh I updated here to filter out IOUs and expense reports, and it worked for me locally. Let me know if this isn't working for you!